templates/plugin/table.conf.epp

```diff
@@ -0,0 +1,27 @@
+<Plugin "table">
+<% $collectd::plugin::table::tables.each |$path, $table| { -%>
+  <Table "<%= $path %>">
+<% unless $table['plugin'] =~ Undef { -%>
```

Comment:
`if $table['plugin']` is the same as `unless $table['plugin'] =~ Undef`. I think the if notation is shorter and better to read?

Reply:
if plugin is set `""`, the `if` version will evaluation to `false` while the unless to `true`.